src/emucore/m6502/src/M6502.cxx

```diff
@@ -81,6 +82,16 @@ void M6502::reset()
   SP = 0xff;
   PS(0x20);
 
+    if (getenv("ALE_RANDOMIZE_CPU")) {
```

Comment:
Please name ale_randomize_cpu for consistency. Also set to false in Defaults.cpp. Also indent << 2.